byway/src/main/webapp/routepage.js

```diff
@@ -67,7 +60,7 @@ function calcRoute() {
     origin:  start,
     destination: end,
     travelMode: 'DRIVING',
-    waypoints:  route.map(waypoint => ({location: waypoint})),
+    waypoints:  route.map(waypoint => ({location: waypoint.name})),
```

Comment:
Can the driving directions system take the placeid as the destination ID?

Reply:
It cannot take placeId, only lat/lng, String or place object. I misspoke about using a Place object, it's really a PlaceResult so I will be using the lat/lng field instead of the String.